gensim/models/wrappers/fasttext.py

```diff
@@ -220,6 +220,10 @@ def save(self, *args, **kwargs):
         super(FastText, self).save(*args, **kwargs)
 
     @classmethod
+    def load_word2vec_format(cls, *args, **kwargs):
+        return FastTextKeyedVectors.load_word2vec_format(*args, **kwargs)
```

Comment:
yes, this method is **not used now** for loading using bin only. I removed this unused code, but got a [strange flake8 error](https://travis-ci.org/RaRe-Technologies/gensim/jobs/244678046) for python 3+, therefore re-added this for this PR. I'll try removing these unused codes later maybe in a different PR. @gojomo